tests/e2e/test_rosacli_node_pool.go

```diff
@@ -859,4 +858,134 @@ var _ = Describe("Edit nodepool",
+
+				By("Get the cluster informations")
+				rosaClient.Runner.JsonFormat()
+				jsonOutput, err := clusterService.DescribeCluster(clusterID)
+				Expect(err).To(BeNil())
+				rosaClient.Runner.UnsetFormat()
+				jsonData := rosaClient.Parser.JsonData.Input(jsonOutput).Parse()
+				clusterName := jsonData.DigString("display_name")
+				clusterProductID := jsonData.DigString("product", "id")
+				var clusterNamePrefix string
+				if jsonData.DigString("domain_prefix") != "" {
+					clusterNamePrefix = jsonData.DigString("domain_prefix")
+				} else {
+					clusterNamePrefix = clusterName
+				}
+				clusterTagsString := jsonData.DigString("aws", "tags")
+				clusterTags := common.PraseTagsFronJsonOutput(clusterTagsString)
+
+				By("Get the managed tags for the nodepool")
+				var managedTags = func(npID string) map[string]interface{} {
+					npLabelName := clusterNamePrefix + "-" + npID
+					managedTags := map[string]interface{}{
+						"api.openshift.com/environment":         OCMEnv,
+						"api.openshift.com/id":                  clusterID,
+						"api.openshift.com/legal-entity-id":     organizationID,
+						"api.openshift.com/name":                clusterName,
+						"api.openshift.com/nodepool-hypershift": npLabelName,
+						"api.openshift.com/nodepool-ocm":        npID,
+						"red-hat-clustertype":                   clusterProductID,
+						"red-hat-managed":                       "true",
+					}
+					return managedTags
+				}
+
+				By("Create a machinepool without tags to the cluster")
+				machinePoolName_1 := common.GenerateRandomName("np-73492", 2)
+				requiredTags := managedTags(machinePoolName_1)
+				if len(clusterTags) > 0 {
+					By("Attach cluster AWS tags")
+					for k, v := range clusterTags {
+						requiredTags[k] = v
+					}
+				}
+				output, err := machinePoolService.CreateMachinePool(clusterID, machinePoolName_1, "--replicas", "3")
+				Expect(err).ToNot(HaveOccurred())
+				Expect(rosaClient.Parser.TextData.Input(output).Parse().Tip()).To(ContainSubstring("Machine pool '%s' created successfully on hosted cluster '%s'", machinePoolName_1, clusterID))
+
+				By("Describe the machinepool")
+				resp, err := machinePoolService.DescribeAndReflectNodePool(clusterID, machinePoolName_1)
+				Expect(err).ToNot(HaveOccurred())
+				machinePool1tags := common.ParseTags(resp.Tags)
+				Expect(err).ToNot(HaveOccurred())
+				for k, v := range requiredTags {
+					Expect(machinePool1tags[k]).To(Equal(v))
+				}
+
+				By("Describe the machinepool in json format")
+				rosaClient.Runner.JsonFormat()
+				jsonOutput, err = machinePoolService.DescribeMachinePool(clusterID, machinePoolName_1)
+				Expect(err).To(BeNil())
+				rosaClient.Runner.UnsetFormat()
+				jsonData = rosaClient.Parser.JsonData.Input(jsonOutput).Parse()
+				tagsString := jsonData.DigString("aws_node_pool", "tags")
+				tags := common.PraseTagsFronJsonOutput(tagsString)
+				for k, v := range requiredTags {
+					Expect(tags[k]).To(Equal(v))
+				}
```

Comment:
Do we need to do this validation? It is not the case to validate the output format. For feature function testing, I think we just need to choose one of `text` and `json`.